assets/js/blocks/products/all-products/block.json

```diff
@@ -0,0 +1,43 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/all-products",
+	"title": "All Products",
+	"category": "woocommerce",
+	"keywords": [ "WooCommerce", "woo-gutenberg-products-block" ],
```

Comment:
Hey, here you don't need that second keyword `"woo-gutenberg-products-block"` in the array, instead you could add need to add it as a `textdomain` like in 👉  [here](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/assets/js/blocks/product-best-sellers/block.json#L94-L96) (also would be great if you could add `apiVersion` and `$schema` properties).

Reply:
Ah, good spot, thank you! I'd accidentally kept that as a keyword 😅 

Anyway, I've removed it in 283e64ab187364ec7a689dcd233290134796ddc3, and added a `textdomain` field in 28561d32905a9cae3645a7f8ba8d6a85f4dd40da.
`$schema` and `apiVersion` have already been around (at the top of the file, rather than the bottom.)

Reply:
Ahh sorry, missed those two keys 😅 Thanks for fixing it!